test/e2e/cmd/run/command.go

```diff
@@ -74,6 +75,7 @@ func Command() *cobra.Command {
 	cmd.Flags().BoolVar(&flags.skipCleanup, "skip-cleanup", false, "Do not run cleanup actions after test run")
 	cmd.Flags().StringVar(&flags.testContextOutPath, "test-context-out", "", "Write the test context to the given path")
 	cmd.Flags().StringVar(&flags.testLicense, "test-license", "", "Test license to apply")
+	cmd.Flags().StringVar(&flags.monitoringSecrets, "monitoring-secrets", "", "Monitoring secrets to use")
```

Comment:
Trying to take a step back, I'm wondering why we don't let the e2e program fetchs directly the secrets it needs from Vault?

Reply:
I would have nothing against that. That's what we do in deployer too.

Reply:
So, I would advocate for doing that.
This would avoid adding another make target and env vars, and also to have the secret written in a file on the disk.

Then, it would be very nice to do the same with the test licenses which are only used by the e2e runner.
Yes, I'm on a quest to reduce the number of env vars we are using to run our e2e tests :)